### DISCUSSION TITLE
What is the right way to set knitr options in a Quarto manuscript project?

### DISCUSSION THREAD

**Discussion by Jiayou-Chao** [quarto-dev/quarto-cli#10080] [Q&A]
### Description

I set the the following knitr options in a Quarto file and they work in a standalone file:
```
knitr:
  opts_chunk: 
    include: false
    echo: false
    message: false
    warning: false
```

However, for the same file in a Quarto manuscript project, these options seem to be ignored no matter whether I set them in the header or in the code block. The generated HTML file always shows codes and messages. I'm not sure whether this is a bug or I should set them somewhere else?

**Comment by mcanouil** [comment]
Please see the documentation:
- https://quarto.org/docs/reference/cells/cells-knitr.html
- https://quarto.org/docs/computations/execution-options.html
- https://quarto.org/docs/computations/r.html
